### PR TITLE
Bump default Ruby version to 3.1.3.

### PR DESCRIPTION
**Pull request by igorpeshansky**
[b/254273286](http://b/254273286).

Depends on https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/504.

### CODE REVIEW

File: config/software/ruby.rb [hunk 1]
```diff
@@ -50,6 +50,16 @@
 dependency "libiconv"
 
 # The SHA256 checksums below are for the *.tar.gz packages from https://www.ruby-lang.org/en/downloads/releases.
+version("3.1.3")      { source sha256: "5ea498a35f4cd15875200a52dde42b6eb179e1264e17d78732c3a57cd1c6ab9e" }
+version("3.1.2")      { source sha256: "61843112389f02b735428b53bb64cf988ad9fb81858b8248e22e57336f24a83e" }
```
Comment (qingling128) [inline config/software/ruby.rb:54]: Do we need these intermediate versions that are not used?

**Review by qingling128 — APPROVED** [review]
LGTM

File: config/software/ruby.rb (hunk 1 above)
Comment (igorpeshansky) [reply config/software/ruby.rb:54]: They don't hurt, and make this file closer to the one in omnibus-software…

**Review by qingling128 — APPROVED** [review]
LGTM

**Pull request merged**